docs/01-app/02-guides/self-hosting.mdx

```diff
@@ -162,6 +162,22 @@ Using a custom cache handler will allow you to ensure consistency across all pod
 >
 > - `revalidatePath` is a convenience layer on top of cache tags. Calling `revalidatePath` will call the `revalidateTag` function with a special default tag for the provided page.
 
+#### Disable background revalidation
+
+When deploying multiple instance of the same Next.js app it could be interesting to decouple the background revalidation for ISR requests to avoid multiple instances revalidating the same page at the same time. This can be achieved by setting the `disableBackgroundRevalidation` option to `true` in your `next.config.js` file.
```

Comment:
```suggestion
When deploying multiple instance of the same Next.js app it could be useful to manually control the background revalidation for ISR requests to avoid multiple instances revalidating the same page at the same time. This can be achieved by setting the `disableBackgroundRevalidation` option to `true` in your `next.config.js` file.
```